client/src/hooks.ts

```diff
@@ -35,8 +35,9 @@ export async function handle({
 export function getSession(event: RequestEvent) {
 	return event.locals.user
 		? {
-				// Only publish public data here.
+				// Only publish public data here. Do not mix it up with event.locals.user
```

Comment:
What does this mean exactly? What should I not mix up?

Reply:
It just means, that probably this will be just a subset of all the available data in _event.locals.user_.
This is publicly available (on the session), while  _event.locals.user_ will contain all data from the user.

I will add reword this to be more concise